packages/runtime/container-runtime/src/garbageCollectionTombstoneUtils.ts

```diff
@@ -6,26 +6,24 @@
 import { ITelemetryGenericEvent } from "@fluidframework/common-definitions";
 import { packagePathToTelemetryProperty } from "@fluidframework/runtime-utils";
 import { MonitoringContext } from "@fluidframework/telemetry-utils";
-import { throwOnTombstoneUsageKey } from "./garbageCollectionConstants";
+import { throwOnTombstoneLoadKey, throwOnTombstoneUsageKey } from "./garbageCollectionConstants";
 
 /**
- * Decides whether or not to send an error event or a generic event for gc tombstone scenarios
- *
- * Adds isSummarizerClient, packagePath, and error to telemetry properties.
+ * Consolidates info / logic for logging when we encounter a Tombstone
  */
 export function sendGCTombstoneEvent(
     mc: MonitoringContext,
-    event: ITelemetryGenericEvent,
-    isSummarizerClient: boolean,
+    event: ITelemetryGenericEvent & { isSummarizerClient: boolean },
```

Comment:
Why remove this as a separate param and add to the event?

Reply:
I was thinking about your feedback in the other PR (similar change), I think you were suggesting to take in all the props directly rather than taking in an event. But then we'd need a generic property bag, and the result will be a longer and longer function signature which is a smell to me.  So I tried to consolidate as much data as possible in the event itself (which also supports arbitrary other props), and only have "top-level" function params for data that needs to be interpreted/transformed.